frontend/src/components/pages/student/AssessmentExperiencePage.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { useContext } from "react";
+import { Box, HStack, Spacer, Text, VStack } from "@chakra-ui/react";
+
+import StudentContext from "../../../contexts/StudentContext";
+import StudentDashboardHeader from "../../assessments/assessment-creation/StudentDashboardHeader";
+import NavButtons from "../../assessments/student-experience/NavButtons";
+import Question from "../../assessments/student-experience/Question";
+import QuestionNumbers from "../../assessments/student-experience/QuestionNumbers";
+import QuestionTitle from "../../assessments/student-experience/QuestionTitle";
+
+const AssessmentExperiencePage = (): React.ReactElement => {
+  const { test, className, currentQuestionIndex } = useContext(StudentContext);
+
+  return (
+    <VStack align="center" flex="1" spacing="8">
+      <StudentDashboardHeader
+        assessmentName={test?.name ?? ""}
+        classroomName={className}
+      />
+      <Box width="90%">
```

Comment:
Wary of the % width for screensize reasons but maybe it's fine

Reply:
see above for test on a larger screen ⬆️. let me know if there's a specific screen size that i should test, or if you thnk there are specific modifications that i should make here!